server/src/main/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteController.java

```diff
@@ -0,0 +1,111 @@
+package com.objectcomputing.checkins.services.checkinnotes;
+
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Delete;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.http.annotation.Put;
+import io.micronaut.security.annotation.Secured;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.micronaut.security.rules.SecurityRule;
+
+import java.net.URI;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.validation.Valid;
+
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+
+
+@Controller("/services/checkin-note")
+@Secured(SecurityRule.IS_ANONYMOUS)
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name = "checkin-note")
+public class CheckinNoteController {
+
```

Comment:
No error handler for CheckinNotesBadArgException

Reply:
After adding this can you rerun your tests on the Controller and make sure they ran the correct amount of tests and that they passed.

Reply:
no wonder tests didn't fail because I missed adding CheckinNotesBadArgException.